sdk/samples/iot/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ cmake_minimum_required (VERSION 3.10)
 
 set(TARGET_NAME "az_iot_samples_common")
```

Comment:
Other than our earlier discussion, why not rename this to `az_iot_samples` instead?